templater/templates/periodics.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
diff all these with what is in eks-distro-prow-jobs right now and update them, i do not think they include some of my later changes. This is really where if we can come up with a way to avoid the copying we might be better off.